api/v1/storagecluster_types.go

```diff
@@ -163,6 +163,14 @@ type StorageDeviceSet struct {
 	// +optional
 	InitialWeight string `json:"initialWeight,omitempty"`
 
+	// PrimaryAffinity is an optional OSD primary-affinity value within the
+	// range [0,1). This value influence the way Ceph's CRUSH selection of
+	// primary OSDs. Lower value reduce performance bottlenecks (especially
+	// on read operations). If not set, default value is 1.
+	// +kubebuilder:validation:Pattern=`^0.[0-9]+$`
+	// +optional
```

Comment:
```suggestion
	// PrimaryAffinity is an optional OSD primary-affinity value within the
	// range [0,1). This value influences Ceph's CRUSH selection of
	// primary OSDs. A lower value reduces the probability that the OSD will
	// be selected. If not set, default value is 1.
	// https://docs.ceph.com/en/latest/rados/operations/crush-map/#primary-affinity
	// +kubebuilder:validation:Pattern=`^0.[0-9]+$`
	// +optional
```

Reply:
@jarrpa Should I re-submit my PR with those changes?

Reply:
Yes